arm-trafficmanager/2017-03-01/swagger/trafficmanager.json

```diff
@@ -0,0 +1,845 @@
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "The Traffic Manager profile.", 
+             "schema": { 
+               "$ref": "#/definitions/Profile" 
+             } 
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Profiles"
+        ],
+        "operationId": "Profiles_CreateOrUpdate",
+        "description": "Create or update a Traffic Manager profile.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager profile."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            },
+            "description": "The Traffic Manager profile parameters supplied to the CreateOrUpdate operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "The created or updated Traffic Manager profile.", 
+             "schema": { 
+               "$ref": "#/definitions/Profile" 
+             } 
+          },
+          "201": { 
+             "description": "The created or updated Traffic Manager profile.", 
+             "schema": { 
+               "$ref": "#/definitions/Profile" 
+             } 
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Profiles"
+        ],
+        "operationId": "Profiles_Delete",
+        "description": "Deletes a Traffic Manager profile.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager profile to be deleted."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile to be deleted."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": ""
```

Comment:
I haven't seen many teams adding schema for delete operation results. I will add one anyways.